volatility3/framework/plugins/windows/filescan.py

```diff
@@ -13,7 +13,7 @@
 class FileScan(interfaces.plugins.PluginInterface):
     """Scans for file objects present in a particular windows memory image."""
 
```

Comment:
I need to revert this. I assumed it was the plugin version. It doesn't need framework 2.0.1, 2.0.0 is fine.